web/src/main/webapp/test/fixtures/tarjontaNimiFixtures.js

```diff
@@ -0,0 +1,25 @@
+function tarjontaNimiFixtures() {
+    var httpBackend = testFrame().httpBackend
+    httpBackend.when('GET', /.*tarjonta-service\/rest\/v1\/haku\/1.2.246.561.29.00000000001/).respond(
+        {
+            "result": {
+                "nimi": {"kieli_fi": "Testihaku"}
+            },
+            "status": "OK"
+        }
+    )
+    httpBackend.when('GET', /.*tarjonta-service\/rest\/v1\/haku\/1.2.246.561.29.00000000002/).respond(
+        {
+          "status": "NOT_FOUND"
+        }
+    )
+    httpBackend.when('GET', /.*tarjonta-service\/rest\/v1\/hakukohde\/1.2.246.561.20.00000000001/).respond(
+        {
+            "result":{
+                "hakukohteenNimet":{"kieli_fi":"Testi hakukohde"}
```

Comment:
Hakukohteella taitaa olla nykyään suomenkielinen nimi pakollinen. Onkohan se ollut alusta lähtien? Jos ei, niin voisi ehkä lisätä nimen resolvaamiseen fall-backin fi->sv->en. Tämmönen on jouduttu tekemään ainakin organisaatioiden nimien osalta jonnekin päin koodia, kun ihmeteltiin miksi nimi jää tyhjäksi.